helm/chart/maesh/values.yaml

```diff
@@ -68,7 +68,9 @@ metrics:
   prometheus:
     enabled: true
 
-smi: false
+smi:
+  deploy: true
```

Comment:
Did we want to deploy this by default? I think we should _not_ deploy by default.

Reply:
If I get this right, this means that on a blank cluster (without any SMI CRDs previously deployed) I would need to set both flags `smi.enable: true` and `smi.deploy: true`  to have maesh running in SMI mode.
To me changing this default to false introduces a risk to end up with users installing maesh in SMI mode without having the CRDs installed :sweat_smile: . Not super good.
What is the risk of redeploying the CRDs in that case ? It should be a noop right ?

Also WDYT of renaming `deploy` to `deploycrds` to make this variable a bit more self explainatory ?

Reply:
Isn't it the other way around? SMI was false, which meant maesh was not started in SMI mode. Now, smi.enable is still false so no behavior has been changed. However, I liked the rename suggestion and adapted that :)